src/app/utils/route-utils.ts

```diff
@@ -7,6 +7,33 @@ import { Network } from '../../types/network'
 import { SearchScope } from '../../types/searchScope'
 import { isStableDeploy } from '../../config'
 import { getSearchTermFromRequest } from '../components/Search/search-utils'
+import { isLayerNotHidden } from '../../types/layers'
+import { isTesting } from './env'
+
+export const fixedNetwork = isTesting
+  ? undefined
+  : (process.env.REACT_APP_FIXED_NETWORK as Network | undefined)
+export const fixedLayer = isTesting ? undefined : (process.env.REACT_APP_FIXED_LAYER as Layer | undefined)
+
+export type ScopeFreedom =
+  | 'network' // We can select only the network
+  | 'layer' // We can select only the layer
+  | 'both' // We can select both network and layer
```

Comment:
Just a nit, but I would name it network-layer or something similar.

Reply:
OK. Done.